### PR TITLE
`Index` → `Subgraphs`

### PR DESCRIPTION
**Pull request by patsissons**
- majority of the changes are pretty straight forward
- i'll note where i had to make a decision
- added redirects for the old routes

### CODE REVIEW

File: goldsky-docs-diagrams.excalidraw [hunk 1]
```diff
@@ -2206,29 +2206,29 @@
...
       "roundness": null,
       "boundElements": [],
-      "updated": 1692642673756,
+      "updated": 1695759241035,
       "link": null,
       "locked": false,
       "fontSize": 28,
       "fontFamily": 2,
-      "text": "Direct data indexing",
+      "text": "Direct indexing",
```
Comment (patsissons) [inline goldsky-docs-diagrams.excalidraw:2225]: updated to be consistent with other naming in the docs (the other changes are just to re-center the text)

**Comment by patsissons** [comment]
looks like i exported the images at too high res (3x), i'll redo them at 2x shortly

File: src/pages/index.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Getting started
 pageTitle: Goldsky - Getting Started
-description: Index and transform on-chain data. Query with GraphQL, SQL, and more.
+description: Index subgraphs and transform on-chain data. Query with GraphQL, SQL, and more.
```
Comment (patsissons) [inline src/pages/index.md:4]: on the fence about this one, could probably use some extra thought

File: src/pages/index.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ Goldsky provides real-time and historical access to data stored on blockchains a
 TODO: Determine which icons to use
 {% /comment %}
 
-{% quick-link title="Index data" icon="installation" href="/indexing" description="Learn how to index your blockchain data." /%}
+{% quick-link title="Indexing subgraphs" icon="installation" href="/subgraphs" description="Learn how to index your blockchain data." /%}
```
Comment (patsissons) [inline src/pages/index.md:15]: IMO this makes sense

File: src/pages/subgraphs/dedicated-subgraphs.md [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
-title: Dedicated Subgraph Indexing
+title: Dedicated Subgraphs
 description: Index your subgraphs - even on private chains - on dedicated infrastructure for improved customizability and performance.
 ---
 
-The quickest way to get started with Goldsky Indexing is with [Shared Subgraph Indexing](/indexing/shared-subgraph-indexing). When you need improved customizability and performance, Goldsky offers Dedicated Subgraph Indexing. As the name suggests, this is a dedicated environment for you and you only, allowing for customization and optimization at both the indexing and querying layers.
+The quickest way to get started with Goldsky Subgraphs is with [Shared Subgraphs](/subgraphs/shared-subgraphs). When you need improved customizability and performance, Goldsky offers Dedicated Subgraphs. As the name suggests, this is a dedicated environment for you and you only, allowing for customization and optimization at both the indexing and querying layers.
 
-**Customized indexing**
+**Customized subgraphs**
```
Comment (patsissons) [inline src/pages/subgraphs/dedicated-subgraphs.md:8]: probably this, but maybe `Customized indexing` is fine as is

File: src/pages/subgraphs/deploying-subgraphs-base.md [hunk 5]
```diff
@@ -30,12 +30,12 @@ For this example we'll use the [BaseGenesisNFT](https://basescan.org/address/0x1
      ]
    }
    ```
-1. Start indexing with:
+1. Start indexing subgraphs with:
```
Comment (patsissons) [inline src/pages/subgraphs/deploying-subgraphs-base.md:33]: IMO makes sense

File: src/pages/subgraphs/deploying-subgraphs-base.md [hunk 6]
```diff
@@ -30,12 +30,12 @@ For this example we'll use the [BaseGenesisNFT](https://basescan.org/address/0x1
...
    ```shell
    goldsky subgraph deploy base-genesis-nft/1.0.0 --from-abi ./base-genesis-nft.json
    ```
 
-You can [learn more about Instant Subgraphs](<(/indexing/instant-subgraphs)>) and how to configure them in our docs.
+You can [learn more about Instant Subgraphs](/subgraphs/instant-subgraphs) and how to configure them in our docs.
```
Comment (patsissons) [inline src/pages/subgraphs/deploying-subgraphs-base.md:38]: fixing bad link, but note that this page is not referenced anywhere

File: src/pages/subgraphs/webhooks.md
Comment (patsissons) [inline src/pages/subgraphs/webhooks.md:1]: does `webhooks` belong under subgraphs?

**Review by radiofreejohn — APPROVED** [review]

File: src/pages/subgraphs/deploying-subgraphs-base.md (hunk 6 above)
Comment (mikenikles) [reply src/pages/subgraphs/deploying-subgraphs-base.md:38]: That was a request from Hemanth via Slack and he shared the link to the page with the relevant parties directly.

File: src/pages/subgraphs/webhooks.md
Comment (mikenikles) [reply src/pages/subgraphs/webhooks.md:1]: It's as good a place as any at the moment.

File: next.config.js [hunk 7]
```diff
@@ -30,15 +30,31 @@ const nextConfig = {
...
+      ],
+      [
+        "/indexing/deploying-subgraphs-base",
+        "/subgraphs/deploying-subgraphs-base",
+      ],
+      ["/indexing/deploying-subgraphs", "/subgraphs/deploying-subgraphs"],
+      ["/indexing/instant-subgraphs", "/subgraphs/instant-subgraphs"],
+      [
+        "/indexing/shared-subgraph-indexing",
+        "/subgraphs/shared-subgraph-indexing",
```
Comment (mikenikles) [inline next.config.js:55]: Looking at the available files, this should probably change.

```suggestion
        "/subgraphs/shared-subgraphs",
```

**Review by mikenikles — APPROVED** [review]
A few changes requested, but approving so my review isn't a bottleneck.

File: next.config.js (hunk 7 above)
Comment (patsissons) [reply next.config.js:55]: nice catch

**Pull request merged**